.cypress/integration/dashboard.spec.js

```diff
@@ -74,12 +74,12 @@ describe('Testing dashboard table', () => {
 
     cy.get('h2.euiTitle').contains('Traces').should('exist');
     cy.contains(' (7)').should('exist');
-    cy.contains('traceGroup.name: HTTP GET').should('exist');
+    cy.contains('traceGroupFields.name: HTTP GET').should('exist');
```

Comment:
I think we need to revert `traceGroup.name` back into `traceGroup` for backward compatibility. Correct me if I am wrong.

Reply:
Oh, so the end state will be:
* traceGroup: string name
* traceGroupFields: collection of other properties
?

Reply:
Yeah. That's what I expect to make it compatible with data-prepper 0.8.x. Need confirm from @joshuali925

Reply:
if we are using `traceGroup` to be the name, the change should just be renaming `traceGroupFields.name` back to `traceGroup` in this PR. But it will still not be compatible with data-prepper that leaves `traceGroup` empty for subspans, because the field meaning is inconsistent